package.json

```diff
@@ -14,7 +14,7 @@
   ],
   "main": "./lib/reportportal-agent",
   "dependencies": {
-    "reportportal-client": "^5.5.0"
+    "reportportal-client": "git://github.com/reportportal/client-javascript.git#master"
```

Comment:
Please revert

Reply:
Reportportal-client V5.5.0 doesn't know about { Events }, so CI is failed. Should I skip this failure?

Reply:
Let's skip this error for now.

Reply:
If we want to skip this error we should skip all unit tests. I have the task with unit tests for jasmine. I can try to fix it as part of this task https://jiraeu.epam.com/browse/EPMRPP-46994

Reply:
Can you just mock failing import for now?

Reply:
I tried, but it didn't work. Ok, I will try better.

Reply:
If won't work, just leave as is.